### PR TITLE
update charting demos to post-refactor

### PR DESCRIPTION
**Pull request by theiliad**

### CODE REVIEW

**Comment by theiliad** [comment]
The deployment failure seems like a permission issue. What do we need to do here?
image

**Comment by alisonjoseph** [comment]
Deployment issue is unrelated to this PR, we're looking into it.

File: package.json [hunk 1]
```diff
@@ -14,8 +14,8 @@
     "format": "prettier --write 'src/**/*.{js,json,css,scss,md,mdx,yaml}'"
   },
   "dependencies": {
-    "@carbon/charts": "^0.16.15",
-    "@carbon/charts-react": "^0.16.15",
+    "@carbon/charts": "^0.16.22",
+    "@carbon/charts-react": "^0.16.22",
```
Comment (emyarod) [inline package.json]: it looks like there have been a few releases since `0.16.22`, would we want to bring in the latest release?

**Comment by theiliad** [comment]
@alisonjoseph should be good to go. I've got some more updates that I'll add in new PRs soon.

**Pull request merged**